frontend/src/TrafficSignDetector.jsx

```
@@ -0,0 +1,189 @@
+                videoRef.current.srcObject = stream;
+            }
+        } catch (err) {
+            setError("Could not access camera: " + err.message);
+            setIsDetecting(false);
+        }
+    };
+
+    const stopCamera = () => {
+        if (videoRef.current && videoRef.current.srcObject) {
+            const tracks = videoRef.current.srcObject.getTracks();
+            tracks.forEach(track => track.stop());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)